playbooks/globalVariables.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Why this change?

Reply:
@cs-suyog-jinturkar `globalVariables.json` is the default file that contains global variables and gets created while playbooks export. However, in these playbooks, there is no global variable used, so this file comes empty.